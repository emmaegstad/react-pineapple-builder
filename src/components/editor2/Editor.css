.Editor {
  display: flex;
  flex-direction: column;
  align-items: center;
  gap: 0.5vh;
  margin-bottom: 2vh;
  padding: 4vh;
  border: 4px solid #ef476f;
  border-radius: 5px;
  background-color: #118ab2;
}

.selectors {
  display: flex;
  gap: 2vw;
  margin-bottom: 1vh;
}
