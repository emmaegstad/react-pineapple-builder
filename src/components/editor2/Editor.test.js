import { render, screen } from '@testing-library/react';
import Editor from './editor';

test('renders the Editor component', () => {
  const container = render(
    <Editor
      bottom="bottom-1"
      bottomCount="0"
      middle="middle-1"
      middleCount="0"
      top="top-1"
      topCount="0"
      catchphrase=""
    />
  );
  expect(container).toMatchSnapshot();
});
