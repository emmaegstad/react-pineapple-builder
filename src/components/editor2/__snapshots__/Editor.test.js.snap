// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Editor component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Editor"
      >
        <div
          class="selectors"
        >
          <div
            class="form-control"
          >
            <label>
              Top 
            </label>
            <select>
              <option
                value="top-1"
              >
                One
              </option>
              <option
                value="top-2"
              >
                Two
              </option>
              <option
                value="top-3"
              >
                Three
              </option>
            </select>
          </div>
          <div
            class="form-control"
          >
            <label>
              Middle 
            </label>
            <select>
              <option
                value="middle-1"
              >
                One
              </option>
              <option
                value="middle-2"
              >
                Two
              </option>
              <option
                value="middle-3"
              >
                Three
              </option>
            </select>
          </div>
          <div
            class="form-control"
          >
            <label>
              Bottom 
            </label>
            <select>
              <option
                value="bottom-1"
              >
                One
              </option>
              <option
                value="bottom-2"
              >
                Two
              </option>
              <option
                value="bottom-3"
              >
                Three
              </option>
            </select>
          </div>
        </div>
        <div
          class="form-control"
        >
          <label>
            Add a Catchphrase! 
          </label>
          <input
            type="text"
            value=""
          />
          <button>
            Add
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Editor"
    >
      <div
        class="selectors"
      >
        <div
          class="form-control"
        >
          <label>
            Top 
          </label>
          <select>
            <option
              value="top-1"
            >
              One
            </option>
            <option
              value="top-2"
            >
              Two
            </option>
            <option
              value="top-3"
            >
              Three
            </option>
          </select>
        </div>
        <div
          class="form-control"
        >
          <label>
            Middle 
          </label>
          <select>
            <option
              value="middle-1"
            >
              One
            </option>
            <option
              value="middle-2"
            >
              Two
            </option>
            <option
              value="middle-3"
            >
              Three
            </option>
          </select>
        </div>
        <div
          class="form-control"
        >
          <label>
            Bottom 
          </label>
          <select>
            <option
              value="bottom-1"
            >
              One
            </option>
            <option
              value="bottom-2"
            >
              Two
            </option>
            <option
              value="bottom-3"
            >
              Three
            </option>
          </select>
        </div>
      </div>
      <div
        class="form-control"
      >
        <label>
          Add a Catchphrase! 
        </label>
        <input
          type="text"
          value=""
        />
        <button>
          Add
        </button>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
